### DISCUSSION TITLE
Link from Saved Search to Custom Record

### DISCUSSION THREAD

**Discussion by markuslille** [burkybang/SuiteAdvanced-Feedback#50] [Q&A] [answered]
I often go back and forth between the record type and the saved search to check my columns and field settings, or to create new fields on my record to build up an advanced search.

It would help me to see a button to the record type while in the edit and view mode of my search.

Thank you.

**Comment by burkybang** [comment, marked as answer]
Here's how you can do this currently. It's 2 clicks.

1. On the results view OR edit view of the saved search, click List at the top right.
![image](https://user-images.githubusercontent.com/927886/197094382-88d70ca6-687a-415d-885f-bcb3e40ace7e.png)

2. On the next page, click Edit Record Type at the top right.
![image](https://user-images.githubusercontent.com/927886/197094396-2eeedd56-dc6c-46d5-ad7c-1d6df90aac32.png)
